Classes/Bookmark/BookmarkStore.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  BookmarkStore.swift
+//  Freetime
+//
+//  Created by Rizwan on 18/10/17.
+//  Copyright © 2017 Ryan Nystrom. All rights reserved.
+//
+
+import Foundation
+
+final class BookmarksStore {
+    static let shared = BookmarksStore()
+    
+    private let archivePath: String = {
+        return FileManager.default
+            .urls(for: .documentDirectory, in: .userDomainMask)
+            // crash intentionally on startup if doesn't exist
+            .first!
```

Comment:
Does this intentional crash ever happen? Or is it just in an eventuality where the app fails for some reason?

Would be nice if we didn't have a known crash 😅

Reply:
I think it *shouldn't* ever. Doing a `fatalError` with a message might be better here, too?

Reply:
Actually, i got inspired from this [line 17](https://github.com/rnystrom/GitHawk/blob/c30156c5a62a9d3899dc898f20ccb3a778bab549/Classes/Notifications/NotificationsDataSource.swift#L17) from `NotificationsDataSource`.

But i agree with doing a `fatalError` with a message.